iphone/Classes/GeolocationModule.m

```diff
@@ -486,7 +486,9 @@ - (void)performGeo:(NSString *)direction address:(NSString *)address callback:(G
     @"mid" : [TiUtils appIdentifier],
     @"sid" : sid,
     @"q" : address,
-    @"c" : [[NSLocale currentLocale] objectForKey:NSLocaleCountryCode]
+    @"c" : [[NSLocale currentLocale] objectForKey:NSLocaleCountryCode] != nil
+        ? [[NSLocale currentLocale] objectForKey:NSLocaleCountryCode]
+        : @"US"
```

Comment:
You can use
```objc
NULL_IF_NIL([[NSLocale currentLocale] objectForKey:NSLocaleCountryCode]);
```
or
```objc
[[NSLocale currentLocale] objectForKey:NSLocaleCountryCode] ?: @"US";
```

Reply:
@AndreaVitale `[[NSLocale currentLocale] objectForKey:NSLocaleCountryCode]` is nullable. So I think we should use -
`NULL_IF_NIL([[NSLocale currentLocale] objectForKey:NSLocaleCountryCode])`, as Hans has suggested,  instead of setting fallback countryCode "US".